### PR TITLE
Feat/remove memory runner

### PR DESCRIPTION
**Pull request by brian-mulier-p**
A lot going on here. I had some troubles to deal with:
- had to populate tables class mapping if not present for memory runner to run out-of-the-box
- indexers weren't all services and the implementation of the MemoryRunner was checking for every queue to be currently polling. I believe it was done because the produce method triggered the consumer so if the receive was not there it would fail and after second-thought it's probably not needed for that change. Still I think it's better to have a blocking run method, we can remove that I don't mind
- server-type was mandatory for plenty of reasons: if not the livenessService bean would not pop up and crash the app. I made it optional as all tests that are memory-runner based don't provide one although for production it might not be ideal idk

Basically:
runner-memory implements memory-repository (includes some auto-configuration) implements h2
on EE runner-memory-ee implements runner-memory (so all the chain above) + h2-ee to embed the migrations from EE

LAST ISSUE FOR TRANSPARENT MEMORY RUNNER:
@MicronautTest(transactional = false) is now needed or the queue won't work so I'm not sure how to deal with that :'(

EDIT:
The solution taken (and after discussion with Ludo) is to create a @KestraTest annotation that will expose and act the same as @MicronautTest but with transactional being `false` by default. This will also go along the Micronaut removal from plugins

### CODE REVIEW

File: core/src/main/java/io/kestra/core/repositories/LocalFlowRepositoryLoader.java [hunk 1]
```diff
@@ -65,12 +68,9 @@ public void load(URL basePath) throws IOException, URISyntaxException {
         }
     }
 
-
     public void load(File basePath) throws IOException {
-        this.load(basePath, false);
-    }
-
-    public void load(File basePath, Boolean update) throws IOException {
+        Map<String, Flow> flowByUidInRepository = flowRepository.findAllForAllTenants().stream()
```
Comment (brian-mulier-p) [inline core/src/main/java/io/kestra/core/repositories/LocalFlowRepositoryLoader.java:72]: I changed the implementation to a bulk fetch and always allow update for tests (to prevent tests from throwing errors due to flow already existing) but I just saw it's used by StandaloneCommand. Idk how critical it is to fetch the whole instance's flows. I can add a boolean to avoid bulk fetch in StandaloneCommand but I'm not sure, wdyt ?

**Comment by brian-mulier-p** [comment]
StandaloneCommand -> 1 MIN CONFIGURABLE
